build.gradle

```diff
@@ -69,46 +70,46 @@ dependencyManagement {
 
 dependencies {
     // iexec
-    compile "com.iexec.common:iexec-common:$iexecCommonVersion"
-    //compile files("../iexec-common/build/libs/iexec-common-${iexecCommonVersion}.jar")
+    implementation "com.iexec.common:iexec-common:$iexecCommonVersion"
+    //implementation files("../iexec-common/build/libs/iexec-common-${iexecCommonVersion}-all.jar")
```

Comment:
Cant we remove that now?

Reply:
I prefer to keep it for the moment until the PR in iexec-common is merged and everyone has synchronized its branches.
I think it's best to delete it at a later date. Some other cleanups should be possible at the same time.